apis/pkg/v1alpha1/configuration_types.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
```

Comment:
Per our [feature lifecycle policy](https://docs.crossplane.io/v1.10/reference/feature-lifecycle/#alpha-features), these new fields can be added to GA API types but explicity marked as alpha 👍🏻